pbs/pbsrequest.go

```diff
@@ -72,7 +72,8 @@ type PBSRequest struct {
 	AccountID     string          `json:"account_id"`
 	Tid           string          `json:"tid"`
 	CacheMarkup   int8            `json:"cache_markup"`
-	SortBids   	  int8            `json:"sort_bids"`
+	SortBids      int8            `json:"sort_bids"`
+	IsDFP         int8            `json:"is_dfp"`
```

Comment:
should it be "max key length" so it's more flexible? (I know i said isdfp, sorry)